src/workloads/query/LimitSkip.yml

```diff
@@ -0,0 +1,151 @@
+SchemaVersion: 2018-07-01
+Owner: "@mongodb/query-execution"
+Description: |
+  This workload measures performance for $limit and $skip when different constants are supplied as
+  arguments.
+
+Keywords:
+- limit
+- skip
+- cache
+- parametrization
+- classic
+- sbe
+
+GlobalDefaults:
+  Database: &Database test
+  Collection: &Collection Collection0
+  DocumentCount: &DocumentCount 1e6
+  Threads: &Threads 1
+  MaxPhases: &MaxPhases 8
+  LimitNumber: &limitNum {^Inc: {start: 100, step: 100}}
```

Comment:
Given that we do Repeat: 1000, the amount of documents, returned by the query will change from 100 too 100 000 during run time.

It feels a little bit too much variety. Maybe leave the default step of 1?

Reply:
Done.